src/main/java/org/codehaus/mojo/buildhelper/ReserveListenerPortMojo.java

```diff
@@ -121,12 +125,17 @@
     @Parameter
     private boolean randomPort;
 
+    @Parameter (defaultValue = "::1 127.0.0.1")
+    private String interfaces;
```

Comment:
please add some javadocs for the plugin documentation explaining the function of this parameter and it's value syntax.